src/classes/BDI_MigrationMappingHelper.cls

```diff
@@ -37,125 +37,34 @@
-                Boolean objectNameIsUsable = String.isNotBlank(parts.objectName);
-
-                // Check if the current help text mapping's object and field name are usable
-                if (fieldNameIsUsable && objectNameIsUsable) {
-
-                    // Pass help text values to a new instance of MappingDetail wrapper class
-                    BDI_HelpTextCMT.MappingDetail newMappingDetail =
-                            new BDI_HelpTextCMT.MappingDetail(parts.objectName,
-                                    sourceField.Label,
-                                    parts.objectAPIName,parts.fieldName, null, null);
-
-                    helpTextFieldMap.put(sourceField.Name, newMappingDetail);
-
-                    // Remove from map once mapped
-                    allDFRS.remove(dataImportFieldName);
-                }
-            }
-        }
-
-        // Add import status and import lookup field values
-        for (String key : helpTextFieldMap.keySet()) {
-            BDI_HelpTextCMT.MappingDetail mappingDetail = helpTextFieldMap.get(key);
-            setImportStatusAndLookupFields(mappingDetail, importStatusAndLookupMap);
-        }
-
-        return helpTextFieldMap;
+    /*******************************************************************************************************
+    * @description Get a list of by Data Import Field Mapping Set by Developer Name
+    *
+    * @param developerName: String name of the Data Import Field Mapping Set
+    * @return List<Data_Import_Field_Mapping_Set__mdt>
+    */
+    public List<Data_Import_Field_Mapping_Set__mdt> getFieldMappingSet(String developerName) {
+        List<Data_Import_Field_Mapping_Set__mdt> sets = [SELECT Id FROM Data_Import_Field_Mapping_Set__mdt WHERE DeveloperName = :developerName];
+        return sets;
     }
 
-    /**
-     * @description Set import lookup and import status fields on the MappingDetail instance
-     * @param  mappingDetail Instance of MappingDetail where fields are to be set
-     * @param  importStatusAndLookupMap Map holding the field values to be set to MappingDetail
-     */
-    private void setImportStatusAndLookupFields(BDI_HelpTextCMT.MappingDetail mappingDetail,
-            Map<String, BDI_HelpTextCMT.ImportStatusAndLookupParts> importStatusAndLookupMap) {
-
-        BDI_HelpTextCMT.ImportStatusAndLookupParts importStatusAndLookupParts =
-                importStatusAndLookupMap.get(mappingDetail.dataImportObjectName);
-
-        mappingDetail.importedRecordField = importStatusAndLookupParts.importLookupFieldName;
-        mappingDetail.importedRecordStatusField = importStatusAndLookupParts.importStatusFieldName;
+    /*******************************************************************************************************
+    * @description Default packaged Object Mapping Set metadata record
+    */
+    public Data_Import_Object_Mapping_Set__mdt getDefaultObjectMappingSet(String developerName) {
```

Comment:
Is this method still used now?  If not maybe we can remove it.

Reply:
Nope! Removed